ouroboros-consensus/src/ouroboros-consensus/Ouroboros/Consensus/Storage/LedgerDB/BackingStore/API.hs

```diff
@@ -0,0 +1,258 @@
+{-# LANGUAGE DataKinds            #-}
+{-# LANGUAGE DerivingVia          #-}
+{-# LANGUAGE FlexibleContexts     #-}
+{-# LANGUAGE NamedFieldPuns       #-}
+{-# LANGUAGE RankNTypes           #-}
+{-# LANGUAGE StandaloneDeriving   #-}
+{-# LANGUAGE UndecidableInstances #-}
+
+-- | The 'BackingStore' is the component of the
+-- 'Ouroboros.Consensus.Storage.LedgerDB.LedgerDB' implementation that stores a
+-- key-value map with the 'LedgerTable's at a specific point on the chain.
+--
+-- Its intended use is for storing data structures from the
+-- 'Ouroboros.Consensus.Ledger.Basics.LedgerState' and update them with
+-- 'Data.Map.Strict.Diff.Diff's produced by executing the Ledger rules.
+--
```

Comment:
Perhaps this section could benefit from some concrete examples as well (If this suggestion makes sense we could do it in a separate PR).

Reply:
Let's leave this for a separate PR

Reply:
I'm unsure whether there exist examples that belong to the BackingStore alone, and not to block application in general.